packages/core/src/Repo.ts

```diff
@@ -233,32 +219,19 @@ export class Repo<T = any> {
    * @returns The updated document
    */
   async applyChanges(documentId: string, changes: A.Change[]) {
-    this.log('apply changes')
     // apply changes to document
     const doc = (await this.rebuildDoc(documentId)) || A.init()
     const newDoc = A.applyChanges(doc, changes)
 
-    // cache the doc
-    this.docCache.set(documentId, newDoc)
-
-    // append changes to this document's history
-    if (changes.length > 0) await this.appendChangeSet({ documentId, changes })
-
-    // save snapshot
-    await this.saveSnapshot(documentId, newDoc)
-
-    // call handlers
-    for (const fn of this.handlers) {
-      await fn(documentId, newDoc)
-    }
+    await this.set(documentId, newDoc, changes)
```

Comment:
This is how this used to work - `applyChanges` did its thing and then passed on the changes to `set` to do the rest - but we were trying to debug something and changed to duplicate the code in both places. Just changing it back here because I don't want to update the same code in two places.